### ISSUE TITLE
Work on `Troubleshooting` chapter

### ISSUE THREAD

**Issue by src4026** [CTF-handbooks/ctf-handbooks.github.io#10]
# What are the things to add to the `Troubleshooting` section?
- Low RAM Mode
- Map related errors
    - ...
- Map is frozen

**Comment by src4026** [comment]
Comment edited for clarity ->
I've edited the description of the issue to add a tiny outline of it. What do you think about this? We can start assigning the tasks as per our convenience..

**Comment by a-blob** [comment]
I added the titles and WIP labels; I guess this structure is fine?
![image](https://github.com/CTF-handbooks/ctf-handbooks.github.io/assets/88883098/d1298c11-c365-49d1-84a1-0bb162d66ce1)
Probably work on "Map is not working" later since that's what we agreed in pms iirc ¯\\\_(ツ)_/¯

**Comment by src4026** [comment]
Sure, also, I've got another suggestion:
At the start of the page, we could include a recap for some of the things mentioned throughout the main handbook.
- Up-to-date client versions
- resolving errors resultant from improperly set team zones, chest zones, flags figuring out what it is,
if some of them aren't explicitly mentioned in the handbook.

We could also add instructions on how to ask for help providing their `debug.txt` (or other files/the folder depending on the situation). 

Edit: Updated the description of the Issue to reflect the topics we have discussed.
